### ISSUE TITLE
Notepad++ leaves mouse in clicked state

### ISSUE THREAD

**Issue by joeyhub** [notepad-plus-plus/notepad-plus-plus#3380]
sometimes when using notepad++ the mouse gets stuck and the only way to fix it is to restart notepad++. It breaks the mouse across all of windows until you close notepad++. I've had this problem on both windows 7 and windows 10 with the 32 bit version and 64 bit version. It has been like that for many months now.

It's a hard bug to replicate/reproduce.  I would suggest at least scanning over the code for anything that hooks windows UI events/input to see if there is anything obvious there,

**Comment by AngryGamer** [comment]
I've never run into this issue using Notepad++. Are you sure it's not a hardware issue with your mouse or usb ports?

**Comment by joeyhub** [comment]
I considered that and did actually try with another mouse, etc.

It's definitely linked to notepad++ somehow but it does also end up screwing up with the entire OS as well.

It might be my imagination seems to have a habit to trigger when notepad++ has a bit of redraw lag and with the splitter bar in the middle. After it happens it keeps hooking onto the splitter bar as well. The only thing that fixes it is closing notepad++.

There's really nothing else special I know off about the system likely to have an impact. I think the only unusual software that might effect input I have is for my K70 RGB keyboard but I doubt it's that.

**Comment by badon** [comment]
I have confirmed this bug. I have experienced it a few times, and Windows becomes unusable. Always, closing Notepad++ solves the problem, so I'm 100% certain the bug is in Notepad++. The mouse button seems to get stuck in Notepad++ and it retains that behavior throughout Windows somehow. I tested on Windows 7. It's a rare bug, but potentially catastrophic because it causes the inability to use Windows. Before I figured out it was Notepad++, I would have to do a hard reset of my PC, losing all my unsaved work, just to regain control over it.
